src/main/java/use_cases/DatabaseManager.java

```diff
@@ -346,6 +346,48 @@ private void insertPostDB(Post post) {
         }
     }
 
+    /**
+     * Deletes the user from the database if the user exists
+     * @param user
+     */
+    public void deleteUser(User user)
+    {
```

Comment:
could we put these curly braces at the end of the previous line to stay consistent with our code style?